scripts/generate_can_wrapper.py

```diff
@@ -117,19 +117,19 @@
 
 
 def msg_packer_function_prototypes(msg_names):
-    return ''.join(['static bool {}_packer(uint8_t* raw, const CANMsgMap* msgMap, size_t &len);\n'.format(msg_name.lower()) for msg_name in msg_names])
+    return ''.join([f"static bool {msg_name.lower()}_packer(uint8_t* raw, const CANMsgMap* msgMap, size_t &len);\n" for msg_name in msg_names])
 
 
 def msg_unpacker_function_prototypes(msg_names):
-    return ''.join(['static bool {}_unpacker(uint8_t* raw, CANMsgMap* msgMap);\n'.format(msg_name.lower()) for msg_name in msg_names])
+    return ''.join([f"static bool {msg_name.lower()}_unpacker(uint8_t* raw, CANMsgMap* msgMap);\n" for msg_name in msg_names])
 
 
 def packer_msg_switch_cases(msg_names):
-    return ''.join(['case CANID::{}:\n\treturn {}_packer(raw, msgMap, len);\n'.format(msg_name.upper(), msg_name.lower()) for msg_name in msg_names])
+    return ''.join([f"case CANID::{msg_name.upper()}:\n\treturn {msg_name.lower()}_packer(raw, msgMap, len);\n" for msg_name in msg_names])
 
 
 def unpacker_msg_switch_cases(msg_names):
-    return ''.join(['case CANID::{}:\n\treturn {}_unpacker(raw, msgMap);\n'.format(msg_name.upper(), msg_name.lower()) for msg_name in msg_names])
+    return ''.join([f"case CANID::{msg_name.upper()}:\n\treturn {msg_name.lower()}_unpacker(raw, msgMap);\n" for msg_name in msg_names])
```

Comment:
these prints may be more readable with multi line strings instead of the `\r` and `\n`s. 

Example:
```
return ''.join([
    f"""case CANID::{msg_name.upper()}:
             return {msg_name.lower()}_unpacker(raw, msgMap);
     """ for msg_name in msg_names])
```